docs/content/stable/architecture/docdb-sharding/cluster-balancing.md

```diff
@@ -0,0 +1,234 @@
+
+{{< /warning >}}
+
+## Background
+
+### Placement policy
+
+A placement policy is a specification of how data and queries should be distributed across the cluster. For example, a typical [replication factor](../../key-concepts/#replication-factor-rf) of 3 (RF-3) placement might specify that there should be one replica of each [tablet](../../key-concepts/#tablet) in three separate regions (for example, `us-west-1a`, `us-east-2a`, and `us-central-1a`), with [leader](../../key-concepts/#tablet-leader) preference towards `us-west-1a`.
+
+The default placement policy is automatically inferred when a cluster is created, but it can be modified at the cluster-level with [yb-admin](../../../admin/yb-admin/), or overridden on a per-table basis with [tablespaces](../../../explore/going-beyond-sql/tablespaces/) (for YSQL clusters).
+
+### Remote bootstrap
+
+The cluster balancer creates new [tablet](../../key-concepts/#tablet) replicas by a process called remote bootstrap. This involves copying data from a [YB-TServer](../../yb-tserver/) hosting an up-to-date [tablet peer](../../docdb-replication/replication/#tablet-peers) (typically, the geographically closest peer) to the destination [YB-TServer](../../yb-tserver/).
+
+### Blacklisting
+
+A [YB-TServer](../../yb-tserver/) can be blacklisted or "leader blacklisted" to prevent it from hosting data or [leaders](../../key-concepts/#tablet-leader), respectively. Blacklists are used to facilitate graceful shutdowns (for example, when performing staged restarts or OS patching on a node).
+
+## Cluster balancing scenarios
+
+Cluster balancing automatically occurs when a cluster is scaled in or out, when there are node outages (for example, a failed node, or a network partition), and after creating or deleting tables and tablets.
+
+### Cluster balancing when scaling
+
+#### Scaling out
+
+When scaling out (adding nodes), the cluster balancer will gracefully move [tablet](../../key-concepts/#tablet) data and [leaders](../../key-concepts/#tablet-leader) to the new nodes, without any application impact. [Tablet leaders](../../key-concepts/#tablet-leader) handle queries, so moving leaders is equivalent to moving where the queries execute.
+
+#### Scaling in
+
+When scaling in, nodes being removed are marked as "blacklisted" (Applicable only to [YugabyteDB Anywhere](../../../yugabyte-platform/) and [YugabyteDB Aeon](/stable/yugabyte-cloud/)). This signals to the cluster balancer that it should:
+
+- Gracefully transfer [leader](../../key-concepts/#tablet-leader)ship off of the node being removed.
+- Re-replicate the data that was on the node to other nodes in the same region.
+
+After all data and queries have moved off of the node, the node can be removed safely.
+
+### Cluster balancing after node outages
+
+On an unplanned node outage, cluster balancing occurs twice: when the node goes down, and if or when it recovers.
```

Comment:
when the node goes down, and if or when it recovers -> first when the node goes down, and again if the node recovers